test/unit/testExecutorOtherName.js

```diff
@@ -0,0 +1,46 @@
+/* eslint max-lines: ["error", 500] */
+
+const initSteps = require('app/core/initSteps');
+const {expect, assert} = require('chai');
+
+describe('ExecutorsWithOtherNames', () => {
+    const steps = initSteps([__dirname + '/../../app/steps/action/', __dirname + '/../../app/steps/ui']);
+
+    describe('pruneFormData', () => {
+
+        it('test that currentNameReason is deleted when executor has no other name', () => {
```

Comment:
These tests can be combined as there is only one conditional in this method we should only need one test to test the returned params.